nix/pkgs/haskell/haskell.nix

```diff
@@ -92,11 +92,12 @@ let
             platforms = lib.platforms.linux;
           };
 
-          plutus-core.components.benchmarks.plutus-core-create-cost-model = {
-            build-tools = r-packages;
-            # Seems to be broken on darwin for some reason
-            platforms = lib.platforms.linux;
-          };
+          #  Temporarily removed due to CI problems
```

Comment:
I don't think this does what you think it does. You want to leave this here.

Reply:
> I don't think this does what you think it does. You want to leave this here.

OK, but it was failing with

```
Setup: Unknown build target 'test:plutus-core-test-cost-model'.
```

Where is that coming from?  I grepped the whole of `plutus-core` and the `haskell.nix` file was the only one that looked relevant.  Admittedly I did get a new and incomprehensible failure with this change.

Reply:
I think your `updateMaterialized` went wrong somehow. Just revert that change, do the update, and it should work.

Reply:
> I think your `updateMaterialized` went wrong somehow. Just revert that change, do the update, and it should work.

That didn't seem to help, but I think I accidentally changed the wrong entry in `haskell.nix` earlier.  I've now changed `plutus-core.components.tests.plutus-core-test-cost-model` to
          `plutus-core.components.benchmarks.plutus-core-test-cost-model` and that's looking promising.

Reply:
Mind you, it's been sitting there for a while with everything passing, apart from this
![CI](https://user-images.githubusercontent.com/2124565/109209189-23448600-77a3-11eb-8eb5-e64008ca48b7.png)

If I click on `details` I get a big green tick, so I'm not sure why it thinks that's not finished.  Let's see what it's saying tomorrow morning.

Reply:
That's about 15 hours now and it's still in the same state.  I'll try to restart it and see if that helps.

Reply:
This worked after a restart.  I'm going to merge it, but I'd appreciate it if you can OK the changes to `haskell.nix` mentioned three messages above this one, @michaelpj .